.gitignore

```diff
@@ -1,8 +1,28 @@
-/**/target
+# Alphabetically, by-category
+
+# Build artifacts
+#################
+# Ours
+/circuits
```

Comment:
nit: this shouldn't exist anymore now with caching?

Reply:
Thanks!